BRANDING.md

```diff
@@ -0,0 +1,55 @@
+ 
+## Acceptable Uses
+
+Given the open nature of Istio, you may use the Istio trademark to refer to the project without prior written permission. Examples of these acceptable references include the following:
+
+-   To refer to the Istio project itself;
+    
+-   To refer to unmodified source code or other files shared by the Istio repositories on GitHub;
+    
+-   To accurately identify that your design or implementation is based on or is for use with the Istio platform.
+    
+-   To discuss the Istio project for educational or informational purposes
+    
+*Examples:*
+
+-   “[Your Product/Offering] for Istio.”
+    
+-   “[XYZ] Conference for Istio Users.”
+    
+-   “Making [XYZ] better with Istio”
+    
+## Usage Guidelines
+
+-   The Istio name may not be used or registered in a manner that would cause confusion as to the Istio project’s affiliation or endorsement.
+    
+-   Don’t use the Istio name as part of your company name, product name, domain name, or social media profile.
+    
+-   Other than as permitted by these guidelines, the Istio name should not be combined with other trademarks, terms, or source identifiers.
+    
+-   Don't remove, distort or alter any element of the Istio Trademarks without permission:
+    
+-   Do not shorten, abbreviate, or change the spelling of the Istio name, for example in mass communication, product names, social media profiles, domain names or event names).
+    
+-   Do not change the color or design elements of the Istio logo, or combine the logo with any other text.
+    
+-   Community websites, social media accounts, or other community-organized properties or organizations should follow the naming convention "[XYZ] for Istio Users" or "[XYZ] for Istio Contributors".
+    
+-   The name "Istio Working Group" can only be used by approved working groups maintaining code in the Istio GitHub organization under Istio Project governance.
+    
+-   If you’re organizing a conference, training, or certification program, you can use the word Istio to refer to the technology that the event or program covers. However, Istio cannot be contained in the title name for your conference or program (such as IstioConf or Certified Istio Administrator) without further explanation. An example of an appropriate usage could be “XYZ Conference for Istio Users” or “XYZ Certification Program for Istio Users” with XYZ being your own brand name. Any related materials should also prominently disclaim any official affiliation with or endorsement by the Istio project.
+    
+-   Don’t use the Istio logo on any website, product UI, or promotional materials without our prior written permission, except as specified above.
+    
+-   Don't use the Istio name or logo to indicate that your design or implementation is compatible with Istio without prior written permission.
```

Comment:
Does this mean I could not publish something like "Grafana dashboard for Istio" or "cert-manager Istio Adapter"?

Reply:
The first one with the "for Istio" would be OK. The second one we'd want to review; the goal being that consumers are clear on what is/isn't part of the official Istio project.

Reply:
Ok, thanks!. The last one was not hypothetical - https://github.com/cert-manager/istio-csr

Reply:
I think that one is OK because it's not a product. (standard IANAL disclaimer applies!) We might want to have their repo call out that it's not an official part of the Istio project and that Istio project can't provide support for it - that's less a trademark issue though and more Github best practices.

Reply:
I don't think the language yet indicates why the cert-manager/istio-csr one is ok. This seems to imply it wouldn't be:

"Don't use the Istio name or logo to indicate that your design or implementation is compatible with Istio"

@thisisnotapril I wonder which of these is your preferred stance?
1. cert-manager/istio-csr is ok without prior permission. We change the language here to give a reason why.
2. cert-manager/istio-csr should have gotten prior permission to use the Istio name, but we think their usage is ok, and we mail them giving them explicit permission, perhaps asking them to add the callouts as you request.

Reply:
This one confuses me. Above we say:
`To accurately identify that your design or implementation is based on or is for use with the Istio platform` is ok. But isn't that a form of being "compatible" which in this sentence we say requires written permission. Is this trying to cover the cases where someone wrote their own code and thinks they're 100% compatible from an API perspective, while the sentence I quoted is more about using our impl/code?

Reply:
The "compatible" bit is referencing those situations where we might have conformance processes. Perhaps we change it to something like "Don't use the Istio name or logo to indicate that your design or implementation is endorsed by, or a part of the Istio project without prior written permission."